stm32-data-gen/src/low_power.rs

```diff
@@ -0,0 +1,62 @@
+use stm32_data_serde::chip::core::peripheral::rcc::StopMode;
+
+use crate::util::RegexMap;
+
+/// Get the stop mode limit for a peripheral based on the MCU and peripheral name.
+/// Determines the lowest possible stop mode when a peripheral is enabled.
+///
+/// Parameters:
+/// - mcu_name: the full name of the MCU (e.g., "STM32WB55RG")
+/// - peripheral: the name of the peripheral (e.g., "USART1")
+pub(crate) fn get_peripheral_stop_mode_override(mcu_name: &str, peripheral: &str) -> Option<StopMode> {
```

Comment:
this is not an "override" anymore if it's the only source of the info.

Reply:
Will change it to `peripheral_stop_mode_info`